lib/pgfinder/config/columns.yaml

```diff
@@ -0,0 +1,46 @@
+# Configuration file for columns in different versions of input files and the desired pgfinder columns
+# Columns used in ftrs v311 file formats that are relevant to the work, others are discarded
+ftrs_311:
+  - Id
+  - apexRetentionTimeMinutes
+  - chargeOrder
+  - apexMwMonoisotopic
+  - maxIntensity
+# Columns used in ftrs v52 file formats that are relevant to the work, others are discarded
+ftrs_52:
+  - Id
+  - apexRetentionTime
+  - charges
+  - mwMonoIsotopicMass
+  - apexIntensity
+maxquant:
+  - index
+  - Retention time
+  - Mass
+# Columns for the theoretical masses data
+theo_masses:
+  - Inferred structure
+  - Theo (Da)
```

Comment:
Are these duplicates from the pgfinder section?

Reply:
Good spot, forgot I'd moved them to be under `pgfinder` thanks.